src/test/java/com/facebook/openwifirrm/modules/ModelerUtilsTest.java

```diff
@@ -77,4 +86,67 @@ void testValidMetric() throws Exception {
 		);
 		assertEquals(0.861, metric, 0.001);
 	}
+
+	@Test
+	void testBasicPreDot11nAggregatedWifiScanEntry() {
+		final long obsoletionPeriodMs = 900000;
+
+		final String apA = "aaaaaaaaaaaa";
+		final String bssidA = "aa:aa:aa:aa:aa:aa";
+		final String apB = "bbbbbbbbbbbb";
+		final String bssidB = "bb:bb:bb:bb:bb:bb";
+		DeviceDataManager deviceDataManager = new DeviceDataManager();
+		deviceDataManager.setTopology(TestUtils.createTopology("test-zone", apA, apB));
+		DataModel dataModel = new DataModel();
+
+		// if there are no scans (no list or empty list), there should be no aggregates
+		dataModel.latestWifiScans.computeIfAbsent(bssidB, k -> new LinkedList<>());
+		assertTrue(ModelerUtils.getAggregatedWifiScans(dataModel, obsoletionPeriodMs, new MeanAggregator()).isEmpty());
+
+		// for an AP, if there is one scan, the aggregate should just be that scan
+		// for the other device, there should be no aggregate
+		WifiScanEntry entryB1 = TestUtils.createWifiScanEntryWithBssid(1, bssidB); // defaults to signal = -60
```

Comment:
don't assume default values in another place, set anything here that you need

Reply:
Hm, I think this may have consequences beyond just this. I definitely assume certain other things in this and other tests (like having/not having ht_oper, vht_oper, etc.). But I thought part of the purpose of this TestUtils functions are to provide a common/default way of doing things that are useful in tests? Should I try not to rely on things done by TestUtils methods?

Reply:
Sorry, I guess this is more of a specific comment for this call. If the test depends on a specific number, that input better be passed in as an argument, or you should reference the constant somewhere. Otherwise, if someone modifies the value in `TestUtils.createWifiScanEntryWithBssid()` it will be impossible to tell why this is breaking a seemingly unrelated test.